.github/PULL_REQUEST_TEMPLATE/HOTFIX.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Hotfix YYYY-MM-DD
+about: When applying a hotfix for the example site
+title: Hotfix YYYY-MM-DD
+labels: hotfix
+assignees: ''
+
+---
+
+- [ ] Start a new hotfix branch:
+```bash
+$ git flow hotfix start YYYY-MM-DD
+```
+- [ ] Make commits to hotfix branch to address any issues that need to be resolved
+- [ ] Publish the hotfix branch:
+```bash
+$ git flow hotfix publish YYYY-MM-DD
+```
+- [ ] Create a PR for the hotfix branch, target it to `main`, have it reviewed, and confirm that all checks pass
```

Comment:
This lets us test and verify the hotfix on a Netlify deploy preview before merging it, without the overhead of an issue or requiring us to merge unreleased library code from `develop` into `main`.